infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -292,11 +292,13 @@ private void handleWrite(RequestMsg req, ChannelHandlerContext ctx, IServerRoute
         // Its not clear that making all holes high priority is the right thing to do, but since
         // some reads will block until a hole is filled this is required (i.e. bypass quota checks)
         // because the requirement is to allow reads, but only block writes once the quota is exhausted
+        final RequestMsg batchProcessorReq;
```

Comment:
`final RequestMsg batchProcessorReq = req`
then no need for `else` condition

Reply:
The SCA issue here is "Avoid reassigning parameters such as 'req'". 

The other option is to make  `batchProcessorReq`  non-final

Reply:
ooh, ok. do need batchProcessorReq to be final?

P.S. I personally don't see a profit in using `final` inside methods. 
It's just a lot of boilerplate code. Inside the method no need for additional immutability. 

So, I'm ok with any choice you'd choose